libs/code-block/src/lib/modules/code-block/prism-a11y.scss

```diff
@@ -4,8 +4,7 @@
  */
 ::ng-deep code[class*='language-'],
 pre[class*='language-'] {
-  color: black;
-  text-shadow: 0 1px white;
+  text-shadow: 0 1px var(--sky-color-text-inverse);
```

Comment:
most the custom properties have fallbacks but not this one,  do we need one here?

Reply:
Just missed it. That was one of the first ones I did and I forgot the fallback. Updated.